### PR TITLE
added disjoint smart graph documentation to release notes new features

### PR DESCRIPTION
**Pull request by hkernbach**

### CODE REVIEW

File: 3.7/release-notes-new-features37.md [hunk 1]
```diff
@@ -204,6 +204,20 @@ traversals and greatly improves performance for such queries.
 SatelliteGraphs are only available in the Enterprise Edition and the
 [ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
 
+DisjointSmartGraphs
+---------------
+
+SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
+A DisjointSmartGraph prohibids connected edges between different SmartGraph
```
Comment (jsteemann) [inline 3.7/release-notes-new-features37.md:211]:
```suggestion
A DisjointSmartGraph prohibits connected edges between different SmartGraph
```

File: 3.7/release-notes-new-features37.md [hunk 2]
```diff
@@ -204,6 +204,20 @@ traversals and greatly improves performance for such queries.
 SatelliteGraphs are only available in the Enterprise Edition and the
 [ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
 
+DisjointSmartGraphs
+---------------
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:208]:
```suggestion
Disjoint SmartGraphs
--------------------
```

File: 3.7/release-notes-new-features37.md [hunk 3]
```diff
@@ -204,6 +204,20 @@ traversals and greatly improves performance for such queries.
...
+SmartGraphs have been extended to a new subtype, called **DisjointSmartGraphs**.
+A DisjointSmartGraph prohibits connected edges between different SmartGraph
+components. In case the graph schema is able to be represented without the need
+of connected SmartGraph components, a DisjointSmartGraph should be used as this
+knowledge can be used by the internal optimizer to gain even better query
+traversal execution times.
+
+
+DisjointSmartGraphs are only available in the Enterprise Edition and the
+[ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
```
Comment (Simran-B) [inline 3.7/release-notes-new-features37.md:219]:
```suggestion
SmartGraphs have been extended with a new option `isDisjoint`.
A Disjoint SmartGraph prohibits edges connecting different SmartGraph
components. If your graph doesn't need edges between vertices with different
SmartGraph attribute values, then you should enable this option. This topology
restriction allows the query optimizer to improve traversal execution times.

Disjoint SmartGraphs are only available in the Enterprise Edition and the
[ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
```

File: 3.7/release-notes-new-features37.md [hunk 4]
```diff
@@ -204,6 +204,18 @@ traversals and greatly improves performance for such queries.
 SatelliteGraphs are only available in the Enterprise Edition and the
 [ArangoDB Cloud](https://cloud.arangodb.com/home?utm_source=docs&utm_medium=cluster_pages&utm_campaign=docs_traffic).
 
+Disjoint SmartGraphs
```
Comment (mchacki) [inline 3.7/release-notes-new-features37.md:207]:
```suggestion
DisjointSmartGraphs
```

File: 3.7/release-notes-new-features37.md [hunk 5]
```diff
@@ -204,6 +204,18 @@ traversals and greatly improves performance for such queries.
...
+--------------------
+
+SmartGraphs have been extended with a new option `isDisjoint`.
+A Disjoint SmartGraph prohibits edges connecting different SmartGraph
```
Comment (mchacki) [inline 3.7/release-notes-new-features37.md:211]:
```suggestion
A DisjointSmartGraph prohibits edges connecting different SmartGraph
```

File: 3.7/release-notes-new-features37.md [hunk 6]
```diff
@@ -204,6 +204,18 @@ traversals and greatly improves performance for such queries.
...
+components. If your graph doesn't need edges between vertices with different
+SmartGraph attribute values, then you should enable this option. This topology
+restriction allows the query optimizer to improve traversal execution times.
+
+Disjoint SmartGraphs are only available in the Enterprise Edition and the
```
Comment (mchacki) [inline 3.7/release-notes-new-features37.md:216]:
```suggestion
DisjointSmartGraphs are only available in the Enterprise Edition and the
```

**Comment by mchacki** [comment]
Ok, i have just read the second PR on this Topic and I am wrong please ignore my Comments
Ready to go from my Point of View.

**Pull request merged**